src/classes/frSetupController.cls

```diff
@@ -90,7 +171,7 @@ public with sharing class frSetupController {
 			}
 		}
 		donationSFOptions.sort();
-		for(frField__mdt field : [SELECT DeveloperName, MasterLabel FROM frField__mdt WHERE Type__c = :DONATION_TYPE ORDER BY MasterLabel]) {
+		for(frField__mdt field : [SELECT DeveloperName, MasterLabel FROM frField__mdt WHERE Type__c = 'Donation' OR Type__c = 'All' ORDER BY MasterLabel]) {
```

Comment:
The ALL type mapping needs to be removed, there isn't a concept of frFields that apply to all objects currently

Reply:
Removed from all queries and as an option on the object